newton/_src/solvers/kamino/_src/core/gravity.py

```
@@ -129,23 +130,23 @@ def acceleration(self, g: float):
         self._acceleration = g
 
     @property
-    def direction(self) -> vec3f:
+    def direction(self) -> wp.vec3f:
         """Returns the normalized direction vector of gravity."""
         return self._direction
 
     @direction.setter
-    def direction(self, dir: vec3f):
+    def direction(self, dir: wp.vec3f):
```

Ruff (0.15.20)

[error] Function argument `dir` is shadowing a Python builtin

(A002)